akka-docs/rst/project/migration-guide-persistence-experimental-2.3.x-2.4.x.rst

```diff
@@ -204,3 +203,67 @@ To continue using LevelDB based persistence plugins it is now required for relat
 to include an additional explicit dependency declaration for the LevelDB artifacts. 
 This change allows production akka deployments to avoid need for the LevelDB provisioning. 
 Please see persistence extension ``reference.conf`` for details. 
+
```

Comment:
um, this migration guide was intended for the changes that we made between 2.3.3 to 2.3.4.
I would guess that most users have already migrated to >= 2.3.4 and this guide is not interesting for them.
Should we put new (2.3.11 -> 2.4.0) changes in the ordinary migration guide.